Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -255,7 +255,7 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
         settings[.SDKROOT] = "auto"
         settings[.SDK_VARIANT] = "auto"
         settings[.SKIP_INSTALL] = "YES"
-        let firstTarget = package.targets.first?.underlyingTarget
+        let firstTarget = package.targets.first(where: { $0.type != .test })?.underlyingTarget ?? package.targets.first?.underlyingTarget
```

Comment:
This ensures we're not using a test target as representative for the package if there are other targets.

Reply:
Hmm, this doesn't seem correct. I think we should set deployment target at target-level now?

Reply:
I think we did this originally because products have no deployment target in the model and the deployment target was the same for all targets anyway. I'm not sure if it is worth taking the full complexity of validating deployment targets for products based on their underlying targets now.

Reply:
but this won't set the right deployment for test targets, right?

Reply:
Yah, that's true, we should special case test targets.